### ISSUE TITLE
Report command runtime

### ISSUE THREAD

**Issue by neilconway** [variadico/noti#35]
It would be convenient to have an option to cause `noti` to report the elapsed runtime of the command that just finished. e.g.,

```
noti make
==> Done after 5:23:26!
```

Happy to send a PR if you agree this would be useful.

**Comment by variadico** [comment]
Hey there.

You can kinda already do this by just adding `time` in there. :)

```
$ noti time make
        1.47 real         1.10 user         0.38 sys
```

**Comment by variadico** [comment]
Fixed by #117.
